src/H5FDfamily.c

```diff
@@ -1082,7 +1082,7 @@ H5FD__family_get_eof(const H5FD_t *_file, H5FD_mem_t type)
      * loop with i==0.
      */
     assert(file->nmembs > 0);
-    for (i = (int)file->nmembs - 1; i >= 0; --i) {
+    for (i = (int)(file->nmembs - 1); i >= 0; --i) {
```

Comment:
What's the reason for this change? It looks like it could create a warning about subtracting a signed number from an unsigned (I assume the (int) was added in the first place to avoid that).

Reply:
It actually removed a warning.  :-)   I can't remember exactly, but one of the ones like "assuming that the range doesn't wrap around".

(It can't actually go negative, since the assert guarantees that nmembs is > 0)